app/views/devise/sessions/new.html.slim

```diff
@@ -16,7 +16,7 @@
         = f.input :remember_me, as: :boolean
 
       .form-actions
-        = f.button :submit, t('app.user.sign_in.title')
+        = f.button :submit, t('.title')
```

Comment:
tha's not a good idea to use page title as button name even if they look similar